### PR TITLE
Add missing "commit_url" to issues events payload

### PR DESCRIPTION
**Pull request by pdalpra**

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Great, thanks 👏 . Could you also add it to https://github.com/github/developer.github.com/blob/master/content/v3/issues/events.md#attributes to be consistent?

**Comment by pdalpra** [comment]
sure :)

**Comment by pdalpra** [comment]
and it's done :)

File: content/v3/issues/events.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ determine who should be notified of comments.
   <dt>commit_id</dt>
   <dd>The String SHA of a commit that referenced this Issue</dd>
 
+  <dt>commit_url</dt>
+  <dd>The Github API link to a commit that referenced this Issue</dd>
```
Comment (mikemcquaid) [inline content/v3/issues/events.md]: `Github` to `GitHub` 😉

Comment (pdalpra) [inline content/v3/issues/events.md]: fixed !

**Comment by mikemcquaid** [comment]
Thanks for your contribution to our developer docs! You rock!

**Comment by pdalpra** [comment]
My pleasure :)

**Pull request merged**